content/docs/programs/immunizations.md

```diff
@@ -0,0 +1,131 @@
++++
+title = "Immunizations"
+description = "Immunization Programs"
+date = 2022-05-17T16:20:00+00:00
+updated = 2022-05-17T16:20:00+00:00
+draft = false
+weight = 62
+sort_by = "weight"
+template = "docs/page.html"
+
+[extra]
+toc = true
+top = false
++++
+
+Choose `Programs` > `Immunizations` in the navigation panel.
```

Comment:
this is highlighting the fact that `patient programs` and `Requisitions` belong in the `programs` menu and are absent!